### PR TITLE
DOC-425 part 6: Pindel, Pindel Annotation, SeSAMe Methylation Beta Estimation, Seurat - 10x Chromium, STAR - Counts, STAR 2-Pass Chimeric, STAR 2-Pass Genome, STAR 2-Pass Transcriptome, STAR-Fusion

### PR DESCRIPTION
**Pull request by ebusygina**
Drafts for Pindel, Pindel Annotation, SeSAMe Methylation Beta Estimation, Seurat - 10x Chromium, STAR - Counts, STAR 2-Pass Chimeric, STAR 2-Pass Genome, STAR 2-Pass Transcriptome, STAR-Fusion workflows

### CODE REVIEW

File: docs/Encyclopedia/pages/Pindel.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Pindel
+
+## Description ##
+
+Pindel is a somatic variant calling pipeline used in GDC whole genome sequencing (WGS), whole exome sequencing (WXS) and targeted sequencing harmonization.
+
+## Overview ##
+
+Pindel is one of the pipelines used for WGS, WXS and targeted sequencing somatic variant calling at the GDC. Somatic variant calling is performed with Pindel using tumor and normal alignments and generates simple nucleotide variation (SNV) data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Pindel.md:9]:
```suggestion
Pindel is one of the pipelines used for WGS, WXS and targeted sequencing somatic variant calling at the GDC. Somatic variant calling is performed with Pindel using tumor and normal alignments and generates indel data.
```

File: docs/Encyclopedia/pages/Pindel_Annotation.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Pindel Annotation
+
+## Description ##
+
+Pindel Annotation is a somatic mutation annotation pipeline used in GDC whole exome sequencing (WXS) and targeted sequencing harmonization.
+
+## Overview ##
+
+Pindel Annotation is a pipeline used for WXS and targeted sequencing somatic mutation annotation at the GDC. Somatic mutation annotation is performed with Pindel Annotation using tumor and normal alignments and generates simple nucleotide variation (SNV) data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Pindel_Annotation.md:9]:
```suggestion
Pindel Annotation is a pipeline used for WXS and targeted sequencing somatic mutation annotation at the GDC. Somatic mutation annotation is performed by annotating raw VCFs, which produces annotated somatic mutation files.
```

File: docs/Encyclopedia/pages/Pindel_Annotation.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### Input
+
+* Tumor VCF - WXS or Targeted Sequencing
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Pindel_Annotation.md:13]:
```suggestion
* Raw VCF - WXS or Targeted Sequencing
```

File: docs/Encyclopedia/pages/Pindel_Annotation.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Normal VCF - WXS or Targeted Sequencing
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Pindel_Annotation.md:14]:
```suggestion
```

File: docs/Encyclopedia/pages/STAR-Fusion.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+STAR-Fusion is a structural variant calling pipeline used in GDC RNA-Seq harmonization.
+
+## Overview ##
+
+STAR-Fusion is a pipeline used for RNA-Seq structural variant calling at the GDC. Structural variant calling is performed with STAR-Fusion using tumor and normal alignments and generates structural variation data.
+
+### Input
+
+* Tumor BAM - RNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR-Fusion.md:13]:
```suggestion
* BAM - RNA-Seq
```

File: docs/Encyclopedia/pages/STAR-Fusion.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+* Normal BAM - RNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR-Fusion.md:14]:
```suggestion
```

File: docs/Encyclopedia/pages/STAR-Fusion.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# STAR-Fusion
+
+## Description ##
+
+STAR-Fusion is a structural variant calling pipeline used in GDC RNA-Seq harmonization.
+
+## Overview ##
+
+STAR-Fusion is a pipeline used for RNA-Seq structural variant calling at the GDC. Structural variant calling is performed with STAR-Fusion using tumor and normal alignments and generates structural variation data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR-Fusion.md:9]:
```suggestion
STAR-Fusion is a pipeline used for RNA-Seq structural variant calling at the GDC. Structural variant calling is performed with STAR-Fusion using tumor or normal alignments and generates structural variation data.
```

File: docs/Encyclopedia/pages/STAR_2-Pass_Chimeric.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# STAR 2-Pass Chimeric
+
+## Description ##
+
+STAR 2-Pass Chimeric is an alignment pipeline used in GDC RNA-Seq harmonization.
+
+## Overview ##
+
+STAR 2-Pass Chimeric is a pipeline used for RNA-Seq reads alignment at the GDC. Alignment is performed with STAR 2-Pass Chimeric using tumor and normal alignments and generates sequencing reads data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR_2-Pass_Chimeric.md:9]:
```suggestion
STAR 2-Pass Chimeric is a pipeline used for RNA-Seq reads alignment at the GDC. Alignment is performed with STAR and generates sequencing reads data.
```

File: docs/Encyclopedia/pages/STAR_2-Pass_Genome.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# STAR 2-Pass Genome
+
+## Description ##
+
+STAR 2-Pass Genome is an alignment pipeline used in GDC RNA-Seq harmonization.
+
+## Overview ##
+
+STAR 2-Pass Genome is a pipeline used for RNA-Seq reads alignment at the GDC. Alignment is performed with STAR 2-Pass Genome using tumor and normal alignments and generates sequencing reads data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR_2-Pass_Genome.md:9]:
```suggestion
STAR 2-Pass Genome is a pipeline used for RNA-Seq reads alignment at the GDC. Alignment is performed with STAR and generates sequencing reads data.
```

File: docs/Encyclopedia/pages/STAR_2-Pass_Transcriptome.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# STAR 2-Pass Transcriptome
+
+## Description ##
+
+STAR 2-Pass Transcriptome is an alignment pipeline used in GDC RNA-Seq harmonization.
+
+## Overview ##
+
+STAR 2-Pass Transcriptome is a pipeline used for RNA-Seq reads alignment at the GDC. Alignment is performed with STAR 2-Pass Transcriptome using tumor and normal alignments and generates sequencing reads data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR_2-Pass_Transcriptome.md:9]:
```suggestion
STAR 2-Pass Transcriptome is a pipeline used for RNA-Seq reads alignment at the GDC. Alignment is performed with STAR and generates sequencing reads data.
```

File: docs/Encyclopedia/pages/STAR_Counts.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+# STAR - Counts
+
+## Description ##
+
+STAR - Counts is an RNA expression pipeline used in GDC RNA-Seq harmonization.
+
+## Overview ##
+
+STAR - Counts is a pipeline used for to quantify RNA gene and exon expression from unharmonized or GDC harmonized data. Quantification is performed with STAR - Counts using tumor and normal alignments and generates transcriptome profiling data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR_Counts.md:9]:
```suggestion
STAR - Counts is a pipeline used for to quantify gene expression from RNA-Seq data. Quantification is performed with STAR using tumor or normal alignments and generates transcriptome profiling data.
```

File: docs/Encyclopedia/pages/STAR_Counts.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+### Input
+
+* Tumor BAM - RNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR_Counts.md:13]:
```suggestion
* Genomic BAM - RNA-Seq
```

File: docs/Encyclopedia/pages/STAR_Counts.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+* Normal BAM - RNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/STAR_Counts.md:14]:
```suggestion
```

File: docs/Encyclopedia/pages/SeSAMe_Methylation_Beta_Estimation.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
+# SeSAMe Methylation Beta Estimation
+
+## Description ##
+
+SeSAMe Methylation Beta Estimation is a workflow used in GDC methylation array harmonization.
+
+## Overview ##
+
+SeSAMe Methylation Beta Estimation is the only pipeline used for methylation array harmonization at the GDC. Methylation array harmonization is performed with SeSAMe Methylation Beta Estimation using tumor and normal alignments and generates dna methylation data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/SeSAMe_Methylation_Beta_Estimation.md:9]:
```suggestion
SeSAMe Methylation Beta Estimation is used for methylation array harmonization at the GDC. Methylation array harmonization is performed with SeSAMe using raw tumor or normal methylation array files and generates beta values and masked intensities data, which removes potential genotype information.
```

File: docs/Encyclopedia/pages/Seurat_10x_Chromium.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
+# Seurat - 10x Chromium
+
+## Description ##
+
+Seurat - 10x Chromium is a secondary gene expression pipeline used in GDC single-cell RNA-Seq (scRNA-Seq) harmonization.
+
+## Overview ##
+
+Seurat - 10x Chromium is a pipeline used for scRNA-Seq for secondary gene expression analysis at the GDC. Secondary gene expression analysis is performed with Seurat - 10x Chromium using tumor alignments and generates transcriptome profiling data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Seurat_10x_Chromium.md:9]:
```suggestion
Seurat - 10x Chromium is a pipeline used for scRNA-Seq for secondary gene expression analysis at the GDC. Secondary gene expression analysis is performed with Seurat using gene expression data and generates transcriptome profiling data.
```

File: docs/Encyclopedia/pages/Seurat_10x_Chromium.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+
+### Input
+
+* Tumor MEX - scRNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Seurat_10x_Chromium.md:13]:
```suggestion
* MEX (Filtered Counts) - scRNA-Seq
```

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Encyclopedia/pages/Pindel.md (hunk 1 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Pindel.md]: changed

File: docs/Encyclopedia/pages/Pindel_Annotation.md (hunk 2 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Pindel_Annotation.md]: changed

File: docs/Encyclopedia/pages/Pindel_Annotation.md (hunk 3 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Pindel_Annotation.md]: changed

File: docs/Encyclopedia/pages/Pindel_Annotation.md (hunk 4 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Pindel_Annotation.md]: deleted

File: docs/Encyclopedia/pages/STAR-Fusion.md (hunk 5 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR-Fusion.md]: changed

File: docs/Encyclopedia/pages/STAR-Fusion.md (hunk 6 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR-Fusion.md]: deleted

File: docs/Encyclopedia/pages/STAR-Fusion.md (hunk 7 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR-Fusion.md]: changed

File: docs/Encyclopedia/pages/STAR_2-Pass_Chimeric.md (hunk 8 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR_2-Pass_Chimeric.md]: changed

File: docs/Encyclopedia/pages/STAR_2-Pass_Genome.md (hunk 9 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR_2-Pass_Genome.md]: changed

File: docs/Encyclopedia/pages/STAR_2-Pass_Transcriptome.md (hunk 10 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR_2-Pass_Transcriptome.md]: changed

File: docs/Encyclopedia/pages/STAR_Counts.md (hunk 11 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR_Counts.md]: changed

File: docs/Encyclopedia/pages/STAR_Counts.md (hunk 12 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/STAR_Counts.md]: changed

File: docs/Encyclopedia/pages/SeSAMe_Methylation_Beta_Estimation.md (hunk 14 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/SeSAMe_Methylation_Beta_Estimation.md]: changed

File: docs/Encyclopedia/pages/Seurat_10x_Chromium.md (hunk 15 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Seurat_10x_Chromium.md]: changed

File: docs/Encyclopedia/pages/Seurat_10x_Chromium.md (hunk 16 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Seurat_10x_Chromium.md]: changed

**Review by wwysoc2 — APPROVED** [review]
LGTM 👍

**Pull request merged**